mediacapture-streams/obtaining-local-multimedia-content/navigatorusermedia/deny.html

```diff
@@ -26,8 +26,13 @@ <h1 class="instructions">Description</h1>
     t.done();
   }),
   t.step_func(function (error) {
-    assert_equals(error.name, "SecurityError", "SecurityError returned");
-    assert_equals(error.constraintName, null, "constraintName attribute not set for permission denied");
+    assert_equals(error.name, "securityError", "securityError returned");
+    // NOTE ALEX: this is also nullable, but practically, all the browsers but FF set it to an empty string.
```

Comment:
Agreed, and the commit you commented reverted to the original code that does just that as far as the error type is concerned. Now, some object are nullable by spec, but the spec does not address the default value at initialization. In that case, my understanding is that it is left to the implementation to decide if an empty string or a null object should be set. Do I understand wrong (and there is a spec that says that e.g. nullable object should be initialized as null)? Should we make the spec explicit in that case?